File: AgentieTurism/AgentieTurism/domain/validator/AgentValidator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AgentieTurism.domain.validator
{
    class AgentValidator : Validator<TravelAgent>
    {
        public void validate(TravelAgent entity)
        {
            if (entity.Passwd.Length <= 6)
                throw new ValidException("Password shoul be at least 6 characters!");
        }
    }
}

File: TourismAgency/AgencyDomain/src/main/java/domain/validators/AgentValidator.java
package domain.validators;

import domain.TravelAgent;

public class AgentValidator implements Validator<TravelAgent> {
    @Override
    public void validate(TravelAgent entity) throws ValidationException {
        if(entity.getPasswd().length()<=6)
            throw new ValidationException("Password shoul be at least 6 characters!");
    }
}
